### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by anatolii-zarytskyi**

### CODE REVIEW

**Review by Iryna-U — APPROVED** [review]

File: src/main/java/mate/academy/spring/controller/UserController.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    public List<UserResponseDto> getAll() {
+        return userService.listUsers()
+                .stream()
+                .map(userMapper::parse)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/{id}")
+    public UserResponseDto get(@PathVariable Long id) {
+        return userMapper.parse(userService.get(id));
```
Comment (Max-Pochepets) [inline src/main/java/mate/academy/spring/controller/UserController.java:35]:
```suggestion
        return userMapper.parseToDto(userService.get(id));
```
we'll have response and request DTOs, so it's good to specify which way you map the object

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]
good job

File: src/main/java/mate/academy/spring/controller/UserController.java (hunk 1 above)
Comment (Anatolii-Zarytskyi) [reply src/main/java/mate/academy/spring/controller/UserController.java]: done

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]
fix travis

**Review by Max-Pochepets — APPROVED** [review]

**Pull request closed without merging**